packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,15 @@
 <template>
   <template if:true={shouldDisplay}>
     <li class={tabClass} title={label} role="presentation">
-      <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
+      <button
+        class="pill__container slds-grid_vertical-align-center slds-p-horizontal_x-small slds-p-vertical_xx-small slds-button slds-var-m-around_xxx-small"
+        onclick={select}
+        aria-label={arialLaelText}
+      >
+        <a class="slds-tabs_default__link" role="tab" onclick={select}
```

Comment:
so we still keep the "role= tab" here?

Reply:
For <a element, it seems attribute "role" here would not have any effect on accessibility behaviour.Is this attribute needed for other features? @erocheleau If, not We should go head and remove attribute "role" from <a.

Reply:
I asked because I saw comment from Etienne & Nathan suggested to remove it, as the tab we use here is not really a 'tab', it's more like a filter, but yes, we have to wait for a double confirmation from @erocheleau ?

Reply:
role=tab removed